src/webview/mod.rs

```diff
@@ -695,6 +697,14 @@ pub trait WebViewBuilderExtWindows {
   ///
   /// Defaults to [`Theme::Auto`] which will follow the OS defaults.
   fn with_theme(self, theme: Theme) -> Self;
+
+  /// Determines whether the custom protocols should use `http://<scheme>.localhost` instead of `http://<scheme>.localhost`.
+  ///
+  /// Using a `http` scheme will prevent mixed content warnings when trying to fetch `http` endpoints
+  /// and is therefore less secure but will match the behavior of the `<scheme>://localhost` protocols on macOS and Linux.
```

Comment:
```suggestion
  /// Using a `http` scheme will allow mixed content when trying to fetch
  /// and is therefore less secure but will match the behavior of the `<scheme>://localhost` protocols on macOS and Linux.
```

Reply:
Maybe "Using a `http` scheme will allow mixed content http requests" or ""Using a `http` scheme will allow mixed content when trying to send http requests" ? It reads a bit awkward cutting off at fetch